tests/Feature/Api/V1/UserRecentlyPlayedGamesTest.php

```diff
@@ -20,6 +20,20 @@ class UserRecentlyPlayedGamesTest extends TestCase
     use RefreshDatabase;
     use BootstrapsApiV1;
 
+    public function testItValidates(): void
+    {
+        $this->get($this->apiUrl('GetUserRecentlyPlayedGames', [
+            'u' => null,
```

Comment:
Is there a difference between explicitly stating null and not providing the parameter?

Reply:
Not in this case, no - added it for brevity. It could also be `''` and should result in the same validation error.

Reply:
Removed it as it seems superfluous now that you mentioned it :)